src/python/pants/engine/internals/native_dep_inference.py

```diff
@@ -16,3 +19,8 @@ class NativeParsedPythonDependencies:
     def __init__(self, imports: dict[str, tuple[int, bool]], string_candidates: dict[str, int]):
         object.__setattr__(self, "imports", FrozenDict(imports))
         object.__setattr__(self, "string_candidates", FrozenDict(string_candidates))
+
+
+def rules() -> Iterable[QueryRule]:
+    # Keep in sync with `intrinsics.rs`.
+    return (QueryRule(NativeParsedPythonDependencies, (NativeDependenciesRequest,)),)
```

Comment:
Ugh. more of these? :unamused:

Reply:
(Not a comment directed at you, these are annoying little thorns. I'm sure it got you as well :|)

Reply:
Nothing exploded when I removed and reran with no pantsd, so I removed this.